demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2920,6 +2924,11 @@ export class DemoMeetingApp
       }
     }
 
+    if (value === 'No Audio') {
+      // An empty media stream destination without any source connected to it
+      return DefaultDeviceController.getAudioContext().createMediaStreamDestination().stream;
```

Comment:
This creates a new `AudioContext` if one not existing, not sure in testing whether you see the audio content been re-used or created. If created do we have to [`close`](https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/close) it to avoid any memory leak? I know this is a demo change but still checking. Will using plain [`MediaStream`](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream/MediaStream) API help here?